plugin/web/extensions/sidebar/core/components/content/common/DatasetCard/Field/Fields/types.ts

```diff
@@ -132,11 +134,22 @@ type PointStroke = {
   }[];
 };
 
+export type ButtonLinkInfo = {
+  title?: string;
+  link?: string;
+};
```

Comment:
I don't think you need to separate this in to its own type. For the ButtonLink, we don't have multiple items or anything, so having these types directly on the ButtonLink type I think is sufficient.

Reply:
done

Reply:
30294f5aa82bcbb0ea19d6d665ee117a42cea95a